akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/JoinConfigCompatCheckerClusterShardingSpec.scala

```diff
@@ -94,6 +94,7 @@ class JoinConfigCompatibilitySpec extends JoinConfigCompatCheckerClusterSharding
 
       val probe = createTestProbe()
       probe.awaitAssert(seedNode.readView.members.size should ===(2), clusterWaitDuration)
+      TestKit.shutdownActorSystem(seedNode.system)
```

Comment:
The seedNode `system` is shutdown after the test has completed by ScalaTestWithActorTestKit, so this should not be needed.